assets/ui.css

```diff
@@ -1,3 +1,14 @@
+.close-button {
+  position: absolute;
```

Comment:
Should we move this to right side? Feels more natural to me. WDYT @saschanaz ?

Reply:
I was asked to add this at that place by @marcoscaceres